orangecontrib/text/widgets/owtwitter.py

```diff
@@ -301,8 +301,8 @@ def search(self):
             if not self.advance:
                 self.api.reset()
             self.search_button.setText("Stop")
-            word_list = self.word_list if self.mode in [self.CONTENT, self.CONTENT_AUTHOR] else []
-            authors = self.word_list if self.mode in [self.AUTHOR, self.CONTENT_AUTHOR] else []
+            word_list = self.word_list if self.mode in [self.CONTENT, self.CONTENT_AUTHOR] else None
```

Comment:
This two lines cause the problems. If `self.word_list` is empty then  word_list  is here set to `['']` instead of `None`.